x-pack/plugins/fleet/server/services/epm/registry/index.ts

```diff
@@ -220,12 +221,16 @@ export async function fetchCategories(
   return fetchUrl(url.toString()).then(JSON.parse);
 }
 
-export async function getInfo(name: string, version: string) {
+export async function getInfo(
+  name: string,
+  version: string,
+  options: { doNotCache?: boolean } = {}
```

Comment:
It's a little confusing to reason about a boolean option with `not` in the name. 

Below, for instance, we have `!options.doNotCache` which is a bit hard to grok. Maybe we could use a "positive" conditional name like `skipCache` or something like that instead.